event-log/src/main/scala/io/renku/eventlog/subscriptions/unprocessed/ProjectPrioritisation.scala

```diff
@@ -28,7 +28,7 @@ import eu.timepit.refined.numeric.NonNegative
 import io.renku.eventlog.EventDate
 import io.renku.eventlog.subscriptions.unprocessed.ProjectPrioritisation.{Priority, ProjectIdAndPath, ProjectInfo}
 
-private class ProjectPrioritisation {
+private[unprocessed] class ProjectPrioritisation {
```

Comment:
I believe if the Prioritisation class needs to private for the `unprocessed` package we could drop `[unprocessed]`

Reply:
We can make it work if we move around the ProjectIdAndPath case class. Maybe in a package